token/client/src/token.rs

```diff
@@ -388,9 +391,9 @@ where
     fn get_multisig_signers<'a, 'b>(
         &self,
         authority: &'b Pubkey,
-        signing_pubkeys: &'a Vec<Pubkey>,
+        signing_pubkeys: &'a [Pubkey],
```

Comment:
also ty for cleaning things like this up, i definitely used vecs in a few places when i started because i was fighting trait impls or the borrowchecker

Reply:
You can actually thank clippy for that! For some reason, it got triggered once there was a change in the implementation.